orders/src/main/java/com/lambdaschool/orders/model/Agents.java

```diff
@@ -0,0 +1,126 @@
+package com.lambdaschool.orders.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "agents")
+public class Agents
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long agentcode;
+    private String agentname;
+    private String workingarea;
+    private double commission;
+    private String phone;
+    private String country;
+
+    @OneToMany(mappedBy = "agent",
+            cascade = CascadeType.ALL,
+            orphanRemoval = true)
+    @JsonIgnoreProperties({"agents"})
+    private List<Customers> customers = new ArrayList<>();
+
+    @OneToMany(mappedBy = "agent",
+            cascade = CascadeType.ALL,
+            orphanRemoval = true)
+    @JsonIgnoreProperties({"agents"})
+    private List<Orders> orders = new ArrayList<>();
+
+    public Agents()
```

Comment:
Line 31 - Use an array of strings, and make sure to both correct the property typo (singular **"agent"**, not plural "agents" - an order has only one agent and `agent` is the property name you gave in `Orders.java`) and include the property "customer" to avoid the unnecessary extra nesting level of printed properties.

To use the array of strings: **`@JsonIgnoreProperties({"agent", "customer"})`**